docs/build.md

```diff
@@ -17,27 +17,38 @@ running Ubuntu 22.04. This will also work with Ubuntu 22.04 under WSL2.
 
 (1) Install the following packages:
 
-```
-$ sudo apt update
-$ sudo apt upgrade
-$ sudo apt install git
-$ sudo apt install build-essential
-$ sudo apt install nasm
-$ sudo apt install clang cmake ninja-build libz-dev libssl-dev pkg-config
+```sh
+sudo apt update
+sudo apt upgrade
+sudo apt install git
+sudo apt install build-essential
+sudo apt install nasm
+sudo apt install clang cmake ninja-build libz-dev libssl-dev pkg-config
 ```
 
 (2) [Install Rust](https://www.rust-lang.org/tools/install).
 
 (3) Add the following Rust magic:
 
-```
-$ rustup default nightly-2025-10-16
-$ rustup component add rust-src --toolchain nightly-2025-10-16-x86_64-unknown-linux-gnu
+```sh
+rustup default nightly-2025-10-16
+rustup component add rust-src --toolchain nightly-2025-10-16-x86_64-unknown-linux-gnu
 ```
 
 (Note: we pin to a specific nightly version for better reproducibility.
 See e.g. [issue 18](https://github.com/moturus/motor-os/issues/18)).
 
+## Clone the Motor OS repo
+
+```sh
+export MOTORH=$HOME/motorh
+mkdir $MOTORH
+cd $MOTORH
+git clone git@github.com:moturus/motor-os.git
```

Comment:
Please use `https://github.com/...`, not `git@github.com:...`: cloning over https works much more reliably.

Reply:
I personally prefer ssh transport over https, as ssh doesn't allow vscode to scum me with signup...
Will change.